sycl/source/detail/scheduler/commands.hpp

```diff
@@ -322,9 +325,10 @@ class Command {
   // synchronous. The only asynchronous operation currently is host-task.
   bool MShouldCompleteEventIfPossible = true;
 
-  /// Indicates that the node will be freed by cleanup after enqueue. Such nodes
-  /// should be ignored by other cleanup mechanisms.
-  bool MPostEnqueueCleanup = false;
+  /// Indicates that the node will be freed by regular graph cleanup. Such nodes
```

Comment:
For my education: do we still have irregular cleanup after this change?

Reply:
The only other cleanup we have is the one during memory object destruction, it removes all the dependency edges associated with that memory object and cleans up the command if it has none left.

Reply:
Sure, that would probably be less misleading.